english/talk_to_neighbour/woman/go-with-scully/talk-aside.md

```diff
@@ -1,2 +1,5 @@
 Scully says "Listen, I need you're help. The world is in grave danger.
-Will you help me?"
\ No newline at end of file
+Will you help me?"
+
+[Help Scully](go-with-scully/help-scully/im-in.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.